nexus/lib/pwscf.py

```diff
@@ -227,6 +227,7 @@ def incorporate_result(self,result_name,result,sim):
                 outdir = os.path.join(self.locdir,c.outdir)
                 command = 'rsync -av {0}/* {1}/'.format(result.outdir,outdir)
                 if not os.path.exists(outdir):
+                    print('Running rsync for the {} directory. This might take a while.'.format(outdir))
                     os.makedirs(outdir)
```

Comment:
Please add a paired printout indicating the completion of rysnc.

Reply:
Done. Also moved the earlier print to just before the rsync. Makes more sense this way.

Reply:
Please add back the printout before copy. They service distinct purpose and there is no harm to be verbose on the status of nexus.